src/Microsoft.Security.Utilities.Core/HighConfidenceMicrosoftSecurityModels/SEC101_101_AzureClientAppLegacyCredential32.cs

```diff
@@ -47,6 +47,12 @@ public override IEnumerable<string> GenerateTestExamples()
             }
         }
 
+        public override IEnumerable<string> GenerateFalsePositiveExamples()
+        {
+            yield return DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss.fffffffZ");
```

Comment:
>DateTime [](http://example.com/codeflow?start=25&length=8)

Someone out there knows of false positives that currently provoke this rule. This one isn't a good example! it doesn't even meet regex length requirements. I'm soliciting an actual repro now.